tests/test_slogpy_console_debug.py

```diff
@@ -0,0 +1,40 @@
+"""The purpose of the code is to give debug and info methods a unit test to make sure that they are working properly"""
```

Comment:
nit: the file has debug in its name, putting other stuff inside it makes it more confusing, remember that "naming is hard" and because it is hard, you will need to spend more than 3 seconds thinking about it.